weblate/trans/machine/weblatetm.py

```diff
@@ -56,11 +94,11 @@ def download_translations(self, source, language, text, unit, user):
         """Download list of possible translations from a service."""
         matching_units = Unit.objects.same_source(unit)
 
-        return [
-            self.format_unit_match(munit, 100)
+        return list(set([
```

Comment:
Why `list(set(` here? The set of units already should be unique (it might be same source, but different project, so this won't reduce it anyway).

Reply:
the example from WeblateOrg/weblate#1791 have shown me, that under certain circumstances the WeblateTranslation offered duplicates - this was my try to remove this duplicates (I am still a python noob)